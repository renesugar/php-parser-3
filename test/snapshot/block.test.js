const parser = require('../main');

describe('block', () => {
  it('single', () => {
    expect(parser.parseEval('{ $var = 1; }')).toMatchSnapshot();
  });
});
