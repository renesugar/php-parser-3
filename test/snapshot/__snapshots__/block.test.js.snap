// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`block single 1`] = `
Program {
  "children": Array [
    Block {
      "children": Array [
        ExpressionStatement {
          "expression": Assign {
            "kind": "assign",
            "left": Variable {
              "byref": false,
              "curly": false,
              "kind": "variable",
              "name": "var",
            },
            "operator": "=",
            "right": Number {
              "kind": "number",
              "value": "1",
            },
          },
          "kind": "expressionstatement",
        },
      ],
      "kind": "block",
    },
  ],
  "errors": Array [],
  "kind": "program",
}
`;
